web-client/integration-tests/petitionsClerkBlockConsolidatedCasesJourney.test.js

```diff
@@ -0,0 +1,108 @@
+import { applicationContext } from '../src/applicationContext';
+import { blockedCasesReportHelper } from '../src/presenter/computeds/blockedCasesReportHelper';
+import { docketClerkConsolidatesCases } from './journey/docketClerkConsolidatesCases';
+import { docketClerkOpensCaseConsolidateModal } from './journey/docketClerkOpensCaseConsolidateModal';
+import { docketClerkSearchesForCaseToConsolidateWith } from './journey/docketClerkSearchesForCaseToConsolidateWith';
+import { docketClerkSetsCaseReadyForTrial } from './journey/docketClerkSetsCaseReadyForTrial';
+import { fakeFile, loginAs, setupTest } from './helpers';
+import { petitionsClerkBlocksCase } from './journey/petitionsClerkBlocksCase';
+import { petitionsClerkCreatesNewCase } from './journey/petitionsClerkCreatesNewCase';
+import { runCompute } from 'cerebral/test';
+import { withAppContextDecorator } from '../src/withAppContext';
+
+const blockedCasesReportHelperComputed = withAppContextDecorator(
+  blockedCasesReportHelper,
+  applicationContext,
+);
+
+let leadDocketNumber;
+let memberCaseDocketNumber;
+
+describe('Manually block consolidated cases', () => {
+  beforeAll(() => {
+    jest.setTimeout(30000);
+  });
+
+  afterAll(() => {
+    cerebralTest.closeSocket();
+  });
+
+  const cerebralTest = setupTest();
+
+  const trialLocation = `Charleston, West Virginia, ${Date.now()}`;
+
+  loginAs(cerebralTest, 'petitionsclerk@example.com');
+  petitionsClerkCreatesNewCase(cerebralTest, fakeFile, trialLocation);
+
+  it('creates lead case', () => {
+    const caseDetail = cerebralTest.getState('caseDetail');
+    cerebralTest.docketNumber = cerebralTest.leadDocketNumber =
+      caseDetail.docketNumber;
+    leadDocketNumber = caseDetail.docketNumber;
+  });
+
+  loginAs(cerebralTest, 'docketclerk@example.com');
+  docketClerkSetsCaseReadyForTrial(cerebralTest);
+  loginAs(cerebralTest, 'docketclerk@example.com');
+
+  loginAs(cerebralTest, 'petitionsclerk@example.com');
+  petitionsClerkCreatesNewCase(cerebralTest, fakeFile, trialLocation);
+
+  it('should set member case docket number', () => {
+    memberCaseDocketNumber = cerebralTest.docketNumber;
+  });
+
+  loginAs(cerebralTest, 'docketclerk@example.com');
+  docketClerkSetsCaseReadyForTrial(cerebralTest);
+
+  loginAs(cerebralTest, 'docketclerk@example.com');
```

Comment:
Is there a need to re-login as a docket clerk again at this moment?

Reply:
Not sure why I originally had the extra logins.  They have been removed.